Makefile

```diff
@@ -506,11 +506,6 @@ promote-staging-manifest: #promote staging manifests to release dir
 	@cp -r manifest_staging/deploy .
 	@rm -rf charts/secrets-store-csi-driver
 	@cp -r manifest_staging/charts/secrets-store-csi-driver ./charts
-	@mkdir -p ./charts/tmp
```

Comment:
Updating the `promote-staging-manifest` makefile target to remove helm package. The helm charts are packaged and published by the GitHub action.